src/uct/cuda/cuda_copy/cuda_copy_md.h

```diff
@@ -18,6 +19,13 @@ typedef enum {
     UCT_CUDA_PREF_LOC_LAST
 } uct_cuda_pref_loc_t;
 
+typedef enum {
+    UCT_CUDA_ALLOC_TYPE_FABRIC,
+    UCT_CUDA_ALLOC_TYPE_PINNED,
+    UCT_CUDA_ALLOC_TYPE_MANAGED,
+    UCT_CUDA_ALLOC_TYPE_LAST = UCT_CUDA_ALLOC_TYPE_MANAGED
+} uct_cuda_alloc_type_t;
```

Comment:
maybe move it to `cuda_cop_md.c`?

Reply:
I prefer to leave it here as it is referenced/used in cuda_ipc transport and having it in the header file makes it convenient.